docs/api/server/CLI-Commands.md

```diff
@@ -0,0 +1,204 @@
+  * [`clean:test`](#clean-commands)
+* Build Commands
+  * [`build`](#build-commands)
+  * [`build:server`](#build-commands)
+  * [`build:bundle`](#build-commands)
+  * [`build:artifacts:statics`](#build-commands)
+  * [`build:prod-sample`](#build-commands)
+  * [`build:sample-modules`](#build-commands)
+* Start Commands
+  * [`start`](#start-commands)
+  * [`start:watch`](#start-commands)
+* Test Commands
+  * [`test`](#test-commands)
+  * [`test:unit`](#test-commands)
+  * [`test:lint`](#test-commands)
+  * [`pretest:integration`](#test-commands)
+  * [`test:integration`](#test-commands)
+  * [`test:danger`](#test-commands)
+  * [`test:git-history`](#test-commands)
+  * [`test:lockfile`](#test-commands)
+  * [`prestart:prod-sample`](#test-commands)
+  * [`start:prod-sample`](#test-commands)
+* Development Commands
+  * [`serve-module`](#test-commands)
+  * [`drop-module`](#test-commands)
+  * [`set-middleware`](#test-commands)
+  * [`set-dev-endpoints`](#test-commands)
+
+## Install Commands
+
+| Command       | Description                                                                                    |
+|---------------|------------------------------------------------------------------------------------------------|
+| `preinstall`  | Runs `check-engines` package to ensure a compatible Node version is used before `npm install`. |
+| `postinstall` | Runs a [`build`](#build-commands) after `npm install`.                                         |
+
+## Clean Commands
+
+| Command       | Description                                                                                    |
+|---------------|------------------------------------------------------------------------------------------------|
+| `clean`  | Executes `clean:build` and `clean:test`. |
+| `clean:build` | Removes folders and files related to [Babel] and [Webpack] builds. |
+| `clean:test` | Removes [Jest] Integration Test Report folder. |
+
+## Build Commands
+
+| Command       | Description                                                                                    |
+|---------------|------------------------------------------------------------------------------------------------|
+| `build` | Using the [`concurrently`] package, runs `build:server` and `build:bundle`. |
+| `build:server` | Executes [Babel] to transpile ES6 Server source files |
+| `build:bundle` | Executes [One App Bundler] to build Browser assets |
+| `build:artifacts:statics` | Runs [Static Assets Script](../../../scripts/build-static-assets-artifact.js) for [Integration Tests] |
+| `build:prod-sample` | Runs `build:sample-modules` and runs the [Build One App Docker Setup Script](../../../scripts/build-one-app-docker-setup.js) for [Integration Tests] |
+| `build:sample-modules` | Builds the [Holocron Modules] in [`prod-sample/sample-modules`] folder for [Integration Tests] |
+
+```
+npm run build:sample-modules -- --archive-built-artifacts --bundle-statics-hostname=https://cdn.example.com
```

Comment:
can you use the [docopt](http://docopt.org/) format throughout?

Reply:
Fixed in a88378aafc426e868db005159f4e769323b6a93d